pkg/sidecar/sidecarpb/slicegateway_handler.go

```diff
@@ -27,32 +27,25 @@ import (
 )
 
 var (
-	sliceGwTc                         = TcInfo{}
-	interClusterDscpClass      string = "Default"
-	dscpClsStringToDscpCodeMap        = map[string]string{
-		"QOS_PROFILE_DSCP_CLASS_DEFAULT": "Default",
-		"QOS_PROFILE_DSCP_CLASS_AF11":    "AF11",
-		"QOS_PROFILE_DSCP_CLASS_AF12":    "AF12",
-		"QOS_PROFILE_DSCP_CLASS_AF13":    "AF13",
-		"QOS_PROFILE_DSCP_CLASS_AF21":    "AF21",
-		"QOS_PROFILE_DSCP_CLASS_AF22":    "AF22",
-		"QOS_PROFILE_DSCP_CLASS_AF23":    "AF23",
-		"QOS_PROFILE_DSCP_CLASS_AF31":    "AF31",
-		"QOS_PROFILE_DSCP_CLASS_AF32":    "AF32",
-		"QOS_PROFILE_DSCP_CLASS_AF33":    "AF33",
-		"QOS_PROFILE_DSCP_CLASS_AF41":    "AF41",
-		"QOS_PROFILE_DSCP_CLASS_AF42":    "AF42",
-		"QOS_PROFILE_DSCP_CLASS_AF43":    "AF43",
-		"QOS_PROFILE_DSCP_CLASS_EF":      "EF",
-	}
-	log = logger.NewLogger()
+	sliceGwTc       = TcInfo{}
+	log             = logger.NewLogger()
+	dscpClassesList = []string{"AF11", "AF12", "AF21", "AF22", "AF23", "AF31", "AF32", "AF33", "AF41", "AF42", "AF43", "EF"}
+	dscpClassInUse  string
 )
 
+func containsDscp(s []string, str string) bool {
+	for _, v := range s {
+		if v == str {
+			return true
+		}
+	}
+	return false
+}
 func sliceGwSetInterClusterDscpConfig(dscpClass string) error {
-	_, valid := dscpClsStringToDscpCodeMap[dscpClass]
-	if !valid {
+	var interClusterDscpClass string = "Default"
```

Comment:
Dont really see a need for this var. The global var dscpClassInUse should be initialized to "Default" instead.